packages/chrome/src/elements/nav/NavItem.tsx

```diff
@@ -13,21 +13,13 @@ import { useNavContext } from '../../utils/useNavContext';
 import { useChromeContext } from '../../utils/useChromeContext';
 
 interface INavItemProps extends HTMLAttributes<any> {
-  /**
-   * Applies product-specific color palette
-   **/
+  /** Sets a product-specific color palette */
   product?: PRODUCT;
-  /**
-   * Indicate which item is current in the nav
-   **/
+  /** Adds the current item to the nav */
   isCurrent?: boolean;
```

Comment:
This prop does not add an item to the nav. It indicates which item in the nav is current. See https://garden.zendesk.com/components/chrome#default for details.

Reply:
Updated in latest push.